### PR TITLE
Allow manual type registration

### PR DESCRIPTION
**Pull request by fubhy**

### CODE REVIEW

File: src/Config/Schema/SchemaConfig.php [hunk 1]
```diff
@@ -22,7 +22,9 @@ public function getRules()
         return [
             'query'    => ['type' => TypeService::TYPE_OBJECT_TYPE, 'required' => true],
             'mutation' => ['type' => TypeService::TYPE_OBJECT_TYPE],
+            'types'    => ['type' => TypeService::TYPE_ARRAY],
             'name'     => ['type' => TypeService::TYPE_STRING],
+
```
Comment (fubhy) [inline src/Config/Schema/SchemaConfig.php]: Please remove this empty line.

File: src/Schema/AbstractSchema.php [hunk 2]
```diff
@@ -10,6 +10,8 @@
 
 
 use Youshido\GraphQL\Config\Schema\SchemaConfig;
+use Youshido\GraphQL\Type\Object\ObjectType;
```
Comment (fubhy) [inline src/Schema/AbstractSchema.php]: These two new use statements are not needed.

**Comment by fubhy** [comment]
@lslinnet Oh and we also need to still add unit tests for this new feature.

**Comment by viniychuk** [comment]
@fubhy I'd like to have an option to add types dynamically, but I haven't decided yet what is the best approach – to create a TypesList-ish class or to add method like "->registerType" to schema.

But for now I'll merge this PR tonight

**Comment by fubhy** [comment]
You mean like a "fake" union type?

Sounds like a good idea since then we can simple to ->getTypesList()->addType(...) etc.

Basically make it a "type bucket" if you want to call it that.

I'd like to wait with merging this PR until we got test coverage and have the code quality issues that I highlighted fixed. Can we wait for @lslinnet to fix that stuff or approve of us fixing it instead?

**Comment by viniychuk** [comment]
I'll wait till tonight, I've already promised to release this before Monday to couple of parties... if @lslinnet doesn't have time I'll add test coverage myself, it's fine. 

And yes, I was thinking about that "types bucket", the only problem, it won't have all types as you add fields. For example `schema->addField(['user' => new UserType])` and then `schema->getTypesList()` would be empty... so maybe we'll call it with prefix/suffix, like "CustomTypes", idk..

**Comment by lslinnet** [comment]
@viniychuk & @fubhy I will not have time to take a look at those test again before Wednesday, so go ahead and push forward with them if need be.

**Pull request merged**